### ISSUE TITLE
[Bug] About the mode.zep

### ISSUE THREAD

**Issue by myprocess** [phalcon/cphalcon#10496]
```php
if this->_exists(metaData, this->getReadConnection()) {
				let this->_errorMessages = [new Message("Record cannot be updated because it does not exist", null, "InvalidUpdateAttempt")];
				return false;
			}
```
about the "this->_exists(metaData, this->getReadConnection())" may be " ! this->_exists(metaData, this->getReadConnection())"

this bug can find on here:https://github.com/phalcon/cphalcon/blob/master/phalcon/mvc/model.zep#L2938

**Comment by qq2868315280** [comment]
this one is the phalcon 1.3.5 c code part:
![phalcon2 0 x 2](https://cloud.githubusercontent.com/assets/12824855/8076133/74478822-0f7b-11e5-9a1d-f2603404170a.png)

this one is the phalcon 2.0.x c code part:
![phalcon1 3 5](https://cloud.githubusercontent.com/assets/12824855/8076129/63af9cd4-0f7b-11e5-9b54-599bd501b584.png)

this one is the phalcon 2.0.x zephir code part:
![phalcon2 0 x](https://cloud.githubusercontent.com/assets/12824855/8076136/77f42cbe-0f7b-11e5-86ef-9c367a53b860.png)

**Comment by sergeyklay** [comment]
Good catch :+1:

**Comment by andresgutierrez** [comment]
This is fixed in 2.0.x branch, thanks

**Comment by myprocess** [comment]
:smile:
